libs/mocks/WordPressMocks/src/main/assets/mocks/mappings/wpcom/stats/stats_search-terms_day.json

```diff
@@ -0,0 +1,35 @@
+{
+    "request": {
+        "method": "GET",
+        "urlPath": "/rest/v1.1/sites/106707880/stats/search-terms/",
+        "queryParameters": {
+            "period": {
+                "equalTo": "day"
+            },
+            "max": {
+                "matches": "[0-9]+"
+            },
+            "date": {
+                "equalTo": "2019-07-17"
```

Comment:
I believe this needs to be `"matches": "(.*)"` as with our other stats requests. Otherwise, I get mocks failures because the request (e.g. `/rest/v1.1/sites/106707880/stats/search-terms/?period=day&max=7&date=2020-01-21&locale=en_US`) doesn't have any matches.

Reply:
For sure – this is fixed!